### DISCUSSION TITLE
DatePicker onFocus fires 2 times

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2607] [Q&A] [answered]
I am using "rsuite/DatePicker" and when i want to handle onFocus, it fires 2 times instead of one.
Just run the code below and tap to datePicker, in console you will se 2 messages. But i expect only one.

```
function App() {
    return <DatePicker onFocus={() => console.log('here')}>dsq</DatePicker>
}
```

**Comment by simonguo** [comment, marked as answer]
It should be a bug, which can be solved temporarily by `target.tagName`.

```tsx
<DatePicker
  onFocus={e => {
    if (e.target.tagName === 'INPUT') {
      console.log('some things');
    }
  }}
/>
```
